lib/DDG/Meta/Data.pm

```diff
@@ -167,4 +169,18 @@ sub _js_callback_name {
     return $fn;
 }
 
+# return fathead data
+sub source_data {
+    my ($self, $source) = @_;
+    my $data = $ia_metadata{source}{$source} if exists $ia_metadata{source}{$source};
+    my @return = (
+        $data->{src_name},
+        $data->{src_domain},
+        $data->{src_options}->{source_info},
+        $data->{src_options}->{is_mediawiki},
+        $data->{perl_module}
+    );
+    return @return;
```

Comment:
It's still a list, just a reference.  You can do 
``` Perl
my @expecting_list = @{DDG::Meta::Data->source_data($src)};
```
if you need it to be a list immediately.